.github/autolabeler.yml

```diff
@@ -0,0 +1,83 @@
+
+# The following paragraph was taken from the Apache Spark project.
+#   see: https://github.com/apache/spark/blob/master/.github/autolabeler.yml
+# Bot page: https://github.com/apps/probot-autolabeler
+# The matching patterns follow the .gitignore spec.
+# See: https://git-scm.com/docs/gitignore#_pattern_format
+# Also, note that the plugin uses 'ignore' package. See also
+# https://github.com/kaelzhang/node-ignore
+
+INFRA:
+  - ".baseline/"
+  - ".github/"
+  - "/dev/"
+  - ".asf.yaml"
+  - ".gitattributes"
+  - ".gitignore"
+  - "baseline.gradle"
+  - "deploy.gradle"
+  - "jitpack.yml"
+  - "site/mkdocs.yml"
+  - "travis.yml"
+BUILD:
+  - "/dev/"
+  - "/project/"
+  - "/assembly/"
+  - "*pom.xml"
+  - "/bin/docker-image-tool.sh"
+  - "/bin/find-spark-home*"
+  - "gradle/"
+  - "**.gradle"
+BASH:
+  - "dev/check-license"
+  - "**.sh"
```

Comment:
The only scripts are in dev, so I think we can remove this tag.

Reply:
I've removed this in favor of just `dev/`. Since PRs can have mutliple labels, I personally still feel that having a check on `**.sh`, which would match any file ending in `.sh`, would be good for future proofing but I will leave the call to your discretion. You've spent far more time than myself looking at PRs and mentally categorizing them.

Additionally, I'll be around to help keep this up to date as time goes on.  😄